disco_aws_automation/disco_group.py

```diff
@@ -182,7 +215,16 @@ def delete_config(self, config_name):
 
     def list_policies(self, group_name=None, policy_types=None, policy_names=None):
         """Returns all autoscaling policies"""
-        return self.autoscale.list_policies(group_name, policy_types, policy_names)
+        asg_policies = self.autoscale.list_policies(group_name, policy_types, policy_names)
+        spot_policies = self._elastigroup_call(
+            self.elastigroup.list_policies,
+            default=[],
+            group_name=group_name,
+            policy_types=policy_types,
+            policy_names=policy_names
+        )
```

Comment:
Why do we need `group_name`, `policy_types`, and `policy_names` for `spot_policies`? This will return the default empty list since Spotinst does not have policies.